ci-operator/config/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main.yaml

```diff
@@ -0,0 +1,39 @@
+binary_build_commands: make build build-vgmanager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+promotion:
+  name: lvm-operator-latest
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.10"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: |
+  go mod vendor
+  mkdir /.cache && chmod 775 -R /.cache ${GOPATH}
```

Comment:
@jarrpa I think you might have face similar permission issues. How did you manage to work around permission issues to create ./cache dir while working on the CI?

Reply:
Best I know we don't:
* CI config: https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-storage/ocs-operator/red-hat-storage-ocs-operator-main.yaml#L82
* Actual tests: https://github.com/red-hat-storage/ocs-operator/blob/main/Makefile#L164